### DISCUSSION TITLE
UserName change Request

### DISCUSSION THREAD

**Discussion by sdmway** [community/community#44733] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi dear GitHub users, i found a user with my dream nickname "clownidze", so i cant use it. Is it somehow possible to request this nickname through github support? because of this i need to add "1337" to my actual username.

Thanks!

**Comment by radar-blip** [comment, marked as answer]
Probably not.
See selected answer here #32361
